src/server/index.tsx

```diff
@@ -95,8 +109,15 @@ server.get('/*', async (req, res) => {
 
            <!-- Styles -->
            <link rel="stylesheet" type="text/css" href="/static/styles/styles.css" />
+           ${userTheme || ''}
+           ${
+             userTheme
+               ? ''
```

Comment:
Maybe forgot to delete something here.

Reply:
Or how about `${userTheme || <link ...`

Reply:
This diff is updated now, moved this stuff into it's own component